src/Driver/ChromeDriver/Downloader.php

```diff
@@ -153,17 +151,17 @@ private function downloadArchive(Driver $driver): string
     private function getBinaryName(Driver $driver): string
     {
         $operatingSystem = $driver->operatingSystem();
-        if ($this->isJsonVersion($driver)) {
+        if (VersionResolver::isJsonVersion($driver->version())) {
             if ($operatingSystem->equals(OperatingSystem::WINDOWS())) {
-                return 'win32/' . self::BINARY_WINDOWS_JSON;
+                return 'win32';
             }
 
             if ($operatingSystem->equals(OperatingSystem::MACOS())) {
-                return 'mac-x64/' . self::BINARY_MAC_JSON;
+                return 'mac-x64';
             }
 
             if ($operatingSystem->equals(OperatingSystem::LINUX())) {
-                return 'linux64/' . self::BINARY_LINUX_JSON;
+                return 'linux64';
```

Comment:
Are you sure this is correct? This was working before, did it not change in some major version again?

Reply:
This is a bit of an odd one tbh. As it's not really a binary you need to resolve anymore for the new api that i've used. But instead you need to specify the platform. From which you can then fetch the URL.